src/DurableTask.AzureStorage/Partitioning/BlobLeaseManager.cs

```
@@ -156,21 +156,24 @@ public async Task CreateLeaseIfNotExistAsync(string partitionId)
             }
             catch (StorageException se)
             {
-                // eat any storage exception related to conflict
-                // this means the blob already exist
-                this.settings.Logger.PartitionManagerInfo(
-                    this.storageAccountName,
-                    this.taskHubName,
-                    this.workerName,
-                    partitionId,
-                    string.Format(
-                        CultureInfo.InvariantCulture,
-                        "CreateLeaseIfNotExistAsync - leaseContainerName: {0}, consumerGroupName: {1}, partitionId: {2}, blobPrefix: {3}, exception: {4}.",
-                        this.leaseContainerName,
-                        this.consumerGroupName,
+                if (se.RequestInformation.HttpStatusCode != 409)
+                {
+                    // eat any storage exception related to conflict
```

small nit: This comment makes a bit more sense above the if statement's conditional imo. I got confused reading it down here, as I assumed it was saying we were eating the exception that we are logging.